guides/common/modules/proc_configuring-project-to-use-ldap.adoc

```diff
@@ -1,13 +1,14 @@
 [id="Configuring_Project_to_Use_LDAP_{context}"]
-= Configuring {ProjectName} to use LDAP
+= Configuring {Project} to use LDAP
 
-In the {ProjectWebUI}, configure {Project} to use LDAP.
+Configure an LDAP authentication source to enable users to log in to {Project} with their existing LDAP credentials.
 
-Note that if you need single sign-on functionality with Kerberos on {ProjectWebUI}, you should use {FreeIPA} and AD external authentication instead.
-For more information, see:
-
-* xref:Using_FreeIPA_{context}[]
-* xref:configuring-active-directory-as-an-external-identity-provider-for-project_{context}[]
+.Prerequisite
+* Your user account has a role assigned that has the following permissions assigned:
+** `view_authenticators`, `create_authenticators`, `edit_authenticators`
+** `view_locations`, `assign_locations`
+** `view_organizations`, `assign_organizations`
```

Comment:
Is this list accurate?

Reply:
We have different procedures for external groups so we'll cover those permissions there. And yes, the point is to focus the scope only on what's documented. So I think we should be okay here; if it turns out it doesn't work this way, we can always revisit.

Thanks!